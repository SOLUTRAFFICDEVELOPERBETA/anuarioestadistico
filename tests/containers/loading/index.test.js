import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import React from 'react';
import LoadingContainer from '../../../containers/loading';

describe('Pruebas en <LoadingContainer/>', () => {
    let wrapper;

    beforeEach(() => {
        wrapper = shallow(<LoadingContainer />)
    })
    test('debe mostrarse correctamente', () => {
        expect(wrapper).toMatchSnapshot();

    })

    test('debe mostrarse correctamente cuando se le pasa el argumento message', () => {
        const message = "prueba de texto"
        wrapper = shallow(<LoadingContainer message={message} />)
        expect(wrapper.find('WithStyles(ForwardRef(Typography))').text().trim()).toBe(message)
       

    })
})