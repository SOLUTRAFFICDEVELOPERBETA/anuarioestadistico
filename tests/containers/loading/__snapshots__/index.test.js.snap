// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <LoadingContainer/> debe mostrarse correctamente 1`] = `
<Container>
  <WithStyles(ForwardRef(Typography))
    align="center"
    color="primary"
    display="block"
    variant="h5"
  >
    Cargando...
  </WithStyles(ForwardRef(Typography))>
  <Spinner />
</Container>
`;
